test/management/jobs.tests.js

```diff
@@ -538,6 +563,58 @@ describe('JobsManager', function() {
     });
   });
 
+  describe('#importUsersJob', function() {
```

Comment:
can you add one to check the old behavior, please? Since it changed above (leftover) and nothing broke, I assume it wasn't tested. The test should assert that the value received back is "an object with the payload inside a property named data" instead of directly "the data object" as the tests below show for the introduced method.

Reply:
Yes added here https://github.com/auth0/node-auth0/pull/492/files#diff-1ec4ac4eb4d7d5873d93bcd0c2173833R310